apps/condo/domains/marketplace/schema/MarketItemPrice.test.js

```diff
@@ -377,5 +392,29 @@ describe('MarketItemPrice', () => {
             })
         })
     })
+
+    test('Scope must be deleted after the price was deleted', async () => {
+        const staffClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+            canReadMarketItems: true,
+            canManageMarketItems: true,
+            canReadMarketItemPrices: true,
+            canManageMarketItemPrices: true,
+            canReadMarketPriceScopes: true,
+            canManageMarketPriceScopes: true,
+        })
+        await createTestOrganizationEmployee(admin, organization, staffClient.user, role)
+        const [marketItem] = await createTestMarketItem(staffClient, marketCategory, organization)
+
+        const [price] = await createTestMarketItemPrice(staffClient, marketItem)
+        const [property] = await createTestProperty(admin, organization)
+        await createTestMarketPriceScope(staffClient, price, property)
+
+        await MarketItemPrice.softDelete(staffClient, price.id)
+
+        const deletedPriceScope = await MarketPriceScope.getAll(admin, { marketItemPrice: { id: price.id } })
+
+        expect(deletedPriceScope).toHaveLength(0)
```

Comment:
Maybe wrap this block with `waitFor` async handler, to be ensure that side effect with soft-delete operation was successfully executed?

 ```suggestion
        await waitFor(async () => {
            const deletedPriceScope = await MarketPriceScope.getAll(admin, { marketItemPrice: { id: price.id } })

            expect(deletedPriceScope).toHaveLength(0)
        })
```

Reply:
This is unnecessary because we delete objects without any tasks.